File: services/cdw-loader/src/main/java/com/researchex/cdwloader/api/ServiceMetadataResponse.java
package com.researchex.cdwloader.api;

import java.time.OffsetDateTime;

/**
 * CDW Loader 서비스 메타데이터 응답 구조.
 */
public record ServiceMetadataResponse(
        String serviceName, String description, OffsetDateTime timestamp) {}

File: services/cdw-loader/src/main/kotlin/com/researchex/cdwloader/api/ServiceMetadataResponse.kt
package com.researchex.cdwloader.api

import java.time.OffsetDateTime

/** CDW Loader 서비스 메타데이터 응답 구조다. */
data class ServiceMetadataResponse(
    val serviceName: String,
    val description: String,
    val timestamp: OffsetDateTime,
)
